.gitignore

```diff
@@ -106,3 +106,6 @@ venv.bak/
 
 # Local configuration
 setupenv.sh
+smswebapp/settings/local.py
+
+.idea
```

Comment:
Could we add a ``# IDE project files`` comment or similar above this to fit in with the rest of the file?